File: back/DevC/src/main/java/com/back/devc/domain/interaction/bookmark/dto/BookmarkedPostResponse.java
package com.back.devc.domain.interaction.bookmark.dto;

import java.time.LocalDateTime;

public record BookmarkedPostResponse(
        long postId,
        String title,
        String authorNickname,
        long categoryId,
        long likeCount,
        long commentCount,
        long viewCount,
        LocalDateTime createdAt
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/interaction/bookmark/dto/BookmarkedPostResponse.kt
package com.back.devc.domain.interaction.bookmark.dto

import java.time.LocalDateTime

data class BookmarkedPostResponse(
    val postId: Long,
    val title: String,
    val authorNickname: String,
    val categoryId: Long,
    val likeCount: Long,
    val commentCount: Long,
    val viewCount: Long,
    val createdAt: LocalDateTime,
    val liked: Boolean,
    val bookmarked: Boolean,
)